wire-ios/Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController.swift

```diff
@@ -364,6 +366,50 @@ final class ConversationViewController: UIViewController {
         updateRightNavigationItemsButtons()
     }
 
+    // MARK: Resolve 1-1 conversations
+
+    private func resolveConversationIfOneOnOne() {
+        guard
+            conversation.messageProtocol != .mls,
+            conversation.conversationType == .oneOnOne,
```

Comment:
If a conversation is a "fake 1:1" (i.e: a group conversation with only two participants), I think it will be filtered out here. Perhaps we should check if the conversation is of type `oneOnOne` **or** if it's a "fake 1:1"

Reply:
Uh so I missed the case of proteus fake 1-1 conversations, that are groups with 2 participants, right? updated the logic, please double check 🙃